src/ensembl/src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice.ts

```diff
@@ -26,7 +26,8 @@ import {
 
```

Comment:
I hate prettier! 😡 I am sure you changed only several lines in this file, but prettier has reformatted half the functions, and it's now hard to find the meaningful changes that you made.

Reply:
Yeah i noticed this as well. Not sure why on my one it is doing this unless someone else prettier wasn't working and hence didnt update correctly.